Yoast/wordpress-seo
# Added help text to Readability Analysis, and shortlinks to expose class.

## Summary

This PR can be summarized in the following changelog entry:

* Added a help text to the Readability Analysis. 

## Relevant technical choices:

* Added shortlink to `class-expose-shortlinks.php`. 

## Test instructions

This PR can be tested by following these steps:

* Check out this branch and build. Go to Readability Analysis.
* There should be a help text, and a working link. 
* Make sure it's in line with the [design example](https://user-images.githubusercontent.com/6073772/43400531-3d53cae0-940e-11e8-9798-3a13ed510f97.png). 

## Quality assurance


Fixes #10595
